CI/batch/docker/README.md

```diff
@@ -0,0 +1,24 @@
+# Updating the Docker Image for AWS Batch
+This is for AutoGluon Devs to update the CI docker environment.
+
+**IMPORTANT**:
+Please push the changes to our github repository even the docker changes will take effect if you don't. This helps to make sure everyone sees the changes you made and build on top.
+
+To update the docker:
+
+- Update the Dockerfile
+- Log into the AWS account that holds the ECR repo on your dev machine.
+- Export the AWS account credentials as environment variables
+- CD to the same folder as the Dockerfile and execute the following:
+
+```shell
+# First export your ecr repo address as a environment variable
+export AWS_ECR_REPO=${your_repo}
```

Comment:
to be clear, is this "our own individual repo" (e.g., to reproduce docker builds) or just one central repo that we all know about but not mentioned here?

Reply:
Yea. This is our aws ecr repo in a specific account. Don't want to put to much detail in a public available readme